src/bidiMapper/domains/events/EventManager.ts

```diff
@@ -324,3 +332,20 @@ export class EventManager implements IEventManager {
     return result.sort((e1, e2) => e1.id - e2.id);
   }
 }
+
+const EVENT_NAMES = new Set([
+  ...Object.values(ChromiumBidi.BrowsingContext.EventNames),
+  ...Object.values(ChromiumBidi.Log.EventNames),
+  ...Object.values(ChromiumBidi.Network.EventNames),
+  ...Object.values(ChromiumBidi.Script.EventNames),
+]);
+
+const checkEventName = (name: string) => {
```

Comment:
Why not `function checkEventName(name: string) { ... `?

https://google.github.io/styleguide/tsguide.html#function-declarations

Reply:
By habit. In the ol' days, arrow functions are faster but nowadays modern browsers treat them both practically equally.